vpr/src/draw/draw.cpp

```diff
@@ -3069,7 +3078,7 @@ static void draw_pin_to_chan_edge(int pin_node, int chan_node, ezgl::renderer* g
     ezgl::rectangle chan_bbox = draw_get_rr_chan_bbox(chan_node);
 
     float x2 = 0, y2 = 0;
-    switch (chan_rr.type()) {
+    switch (rr_graph.node_type(RRNodeId(chan_node))) {
```

Comment:
could we re-use `channel_type` from above ?

In general it is harder to follow the code if this is a return value of a call to an object method that we pass a type-cast input value vs. a simple variable with a name. And if it is the same name we use everywhere and that value is made `const` it is much easier for someone reading the code (as it has human-understandable names and it is clear that the constant  values don't change in-between).

Reply:
This has also been resolved.